__tests__/browserslist.config.test.mjs

```diff
@@ -0,0 +1,21 @@
+/*
+ * SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ */
+
+import browserslist from 'browserslist'
+import { getUserAgentRegex } from 'browserslist-useragent-regexp'
+import { test } from 'node:test'
+import browserslistConfig from '../browserslist.config.js'
+
+test('browserslist.config.js is a valid browserslist config', () => {
+	// Generate a regex that matches user agents to detect incompatible browsers
+	const supportedBrowsersRegExp = getUserAgentRegex({ allowHigherVersions: true, browsers: browserslistConfig })
+	const supportedBrowsers = browserslist(browserslistConfig)
+
+	// Log the results
+	console.log('Supported browsers regex for Nextcloud Web:\n', supportedBrowsersRegExp)
+	console.log('Supported browsers for Nextcloud Web:\n', supportedBrowsers.join(', '))
```

Comment:
@skjnldsv Just curious, did you log it just for checking the config for validity, or to use the logged value somewhere?